x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -561,23 +580,44 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
       );
     }
 
-    if (showViewSeriesLink === true && anomaly.isTimeSeriesViewRecord === true) {
-      items.push(
-        <EuiContextMenuItem
-          key="view_series"
-          icon="visLine"
-          onClick={() => {
-            closePopover();
-            viewSeries();
-          }}
-          data-test-subj="mlAnomaliesListRowActionViewSeriesButton"
-        >
-          <FormattedMessage
-            id="xpack.ml.anomaliesTable.linksMenu.viewSeriesLabel"
-            defaultMessage="View series"
-          />
-        </EuiContextMenuItem>
-      );
+    if (showViewSeriesLink === true) {
+      if (anomaly.isTimeSeriesViewRecord) {
+        items.push(
+          <EuiContextMenuItem
+            key="view_series"
+            icon="visLine"
+            onClick={() => {
+              closePopover();
+              viewSeries();
+            }}
+            data-test-subj="mlAnomaliesListRowActionViewSeriesButton"
+          >
+            <FormattedMessage
+              id="xpack.ml.anomaliesTable.linksMenu.viewSeriesLabel"
+              defaultMessage="View series"
+            />
+          </EuiContextMenuItem>
+        );
+      }
+
+      if (anomaly.isGeoRecord === true) {
+        items.push(
+          <EuiContextMenuItem
+            key="view_in_maps"
+            icon="logoMaps"
```

Comment:
For consistency with the other items in the context menu, I think you should use `gisApp` here.

![image](https://user-images.githubusercontent.com/7405507/166443748-e7b70c9a-ce6c-4f66-90df-18508c42a445.png)

Reply:
Changed in 1a4013bf62e01b1d3f5cde8fd6454843de4ccb2e